ruby/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission = true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] Please add initialization value for `id` because now if created a new person the id will be nil.
[Hint] You can make it like that:
```ruby
@id = Random.rand(1..1000)
```